### DISCUSSION TITLE
Termux Docker and x11-packages

### DISCUSSION THREAD

**Discussion by JesusChapman** [termux/termux-packages#10467] [Q&A] [answered]
Hello, I am currently in Manjaro, I want to continue working with packages for the termux application, but I usually work mostly with x11 packages, I already tried many ways and I can not start those applications, I also tried to install tigervnc and when running vncserver it gives me an invalid ip address. Any ideas?

![Captura desde 2022-05-01 16-09-47](https://user-images.githubusercontent.com/64093255/166394077-86a56548-bb91-4dac-9f40-cce159a1ec5a.png)

**Comment by ghost** [comment, marked as answer]
If you're running VNC server within Termux docker. you'll need to publish port `5901` or similar in docker by passing flag `--publish 5901:5901` in order to expose it to outside world
https://docs.docker.com/config/containers/container-networking/#:~:text=Published%20ports,%2D%2Dpublish%20or%20%2Dp%20flag.

I'm not sure if this is what you need because you didn't post any logs about invalid ip address. making it difficult to guess

**Comment by JesusChapman** [comment]
@wmcbtech30 

![image](https://user-images.githubusercontent.com/64093255/166565214-e610fdea-3bb3-4a8a-9f56-a24a38132eed.png)


that ip address is invalid

**Reply by ghost** [reply]
Then you may have to publish the VNC port to expose it outside container. publishing the port you would use the container's ip address or localhost
